src/doc_scan_service/session/retrieve/identity.profile.report.schemes.compliance.response.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+const Validation = require('../../../yoti_common/validation');
+
+class identityProfileReportSchemesComplianceResponse {
+  constructor(schemesCompliance) {
+    Validation.isPlainObject(schemesCompliance.scheme, 'scheme');
+    this.scheme = schemesCompliance.scheme;
+
+    Validation.isBoolean(schemesCompliance.requirements_met, 'requirements_met');
+    this.requirementsMet = schemesCompliance.requirements_met;
+
+    Validation.isString(schemesCompliance.requirements_not_met_info, 'requirements_not_met_info');
+    this.requirementsNotMetInfo = schemesCompliance.requirements_not_met_info;
```

Comment:
It seems this one should be optional, cause in case the requirements are met, then there is not much to say, right ?

Reply:
yep, updated it to be optional.